iroh-blobs/src/util/local_pool.rs

```diff
@@ -270,14 +270,13 @@ impl LocalPool {
         // we assume that there are exactly as many threads as there are handles.
         // also, we assume that the threads are still running.
         for _ in 0..self.threads_u32() {
-            println!("sending shutdown message");
             // send the shutdown message
             // sending will fail if all threads are already finished, but
             // in that case we don't need to do anything.
             //
             // Threads will add a permit in any case, so await_thread_completion
             // will then immediately return.
-            self.send.send(Message::Finish).ok();
+            self.send.send_blocking(Message::Finish).ok();
```

Comment:
why is this not using an `async` send?

Reply:
I guess that was back when this was used in Drop... It should be async, really...